services/streammanager/bqstream/bqstreammanager_test.go

```diff
@@ -0,0 +1,62 @@
+package bqstream
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	mock_logger "github.com/rudderlabs/rudder-server/mocks/utils/logger"
+)
+
+func TestTimeout(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	mockLogger := mock_logger.NewMockLoggerI(mockCtrl)
+	mockLogger.EXPECT().Errorf(gomock.Any(), gomock.Any()).AnyTimes()
+	pkgLogger = mockLogger
+
+	credentialsFile := os.Getenv("TEST_BQ_CREDENTIALS_FILE")
```

Comment:
Is this env (TEST_BQ_CREDENTIALS_FILE) set somewhere? Else, test will always be skipped right?

Reply:
yes, it refers to a credentials file for our test Google Cloud account